src/js/tabs/osd.js

```diff
@@ -1158,7 +1159,8 @@ OSD.constants = {
     },
     UNIT_TYPES: [
         'IMPERIAL',
-        'METRIC'
+        'METRIC',
+        'BRITISH',
```

Comment:
No needed comma , at the final line.

Reply:
Is better to add taking comma. In this way if you add more elements the diff will show only the latest line, if not it will show two different lines like this.

Reply:
Now I made Travis unhappy. But I don't understand why?

Reply:
@McGiverGim but without a final comma the logic would not be correct? does this make it jump to the next parameter, when there is no parameter?
@Haslinghuis i think Travis fails are not related to your code

Reply:
@Asizon no, it is supported and ignores this trailing comma: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

@haslinghuis travis fails from time to time, ignore it. When you push a new commit it will try to build it again... maybe my earlier suggestion of divide the line into two:
```javascript
 return `${FONT.symbol(SYM.ALTITUDE)}399.7${FONT.symbol(osd_data.unit_mode === 0 ? SYM.FEET : SYM.METRE)}`;
```
to
```javascript
const unit = FONT.symbol(osd_data.unit_mode === 0 ? SYM.FEET : SYM.METRE);
return `${FONT.symbol(SYM.ALTITUDE)}399.7${unit}`;
```

Reply:
Very useful info, thanks @McGiverGim

Reply:
ECMAScript 2017 even allows trailing comma's for parameter definition within a function definition.
To me it is confusing syntax and makes debugging harder, but we have to be compliant to the standards to prevent SonarCloud code smells.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

Reply:
Modified :-)